### PR TITLE
chore: remove remaining website

### PR DESCRIPTION
**Pull request by sxzz**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
I think we can now officially say goodbye to webpack

**Comment by sxzz** [comment]
P.S: `clipboard-copy` should be moved to `docs`.

**Comment by JeremyWuuuuu** [comment]
> P.S: `clipboard-copy` should be moved to `docs`.

I think we can utilize `vueuse` later

File: README.md [hunk 1]
```diff
@@ -87,18 +87,11 @@ the project will install all dependencies and run `lerna bootstrap` to initializ
 With command
 
 ```bash
-$ yarn website-dev
+$ yarn docs:serve
```
Comment (JeremyWuuuuu) [inline README.md:90]: I think this should be `yarn docs:dev`

**Review by jw-foss — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JeremyWuuuuu) [reply README.md:90]: Please address this comment before merged.

Comment (sxzz) [reply README.md]: Done.

**Comment by sxzz** [comment]
@JeremyWuuuuu

Some questions:
- Why not add `docs` to `package.json` `workspaces` field?
   Then, we doesn't need to execute `cd docs && yarn`.
- Should we move some files about building docs to `docs` directory?
	- crowdin-credentials.ts
	- crowdin-generate.ts
	- build-indices.ts
	- and so on...

**Comment by JeremyWuuuuu** [comment]
> @JeremyWuuuuu
> 
> Some questions:
> 
> * Why not add `docs` to `package.json` `workspaces` field?
>   Then, we doesn't need to execute `cd docs && yarn`.
> * Should we move some files about building docs to `docs` directory?
>   
>   * crowdin-credentials.ts
>   * crowdin-generate.ts
>   * build-indices.ts
>   * and so on...

We can optimize that. I think you are right.

**Pull request merged**